content/influxdb/v2.0/query-data/common-queries/count_unique_values_for_column.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Count unique values for a column
+seotitle: Count unique values for a column
+description: >
+  .
+influxdb/v2.0/tags: [queries]
+menu:
+  influxdb_2_0:
+    name: Count unique values
+    parent: Common queries
+weight: 104
+---
+
+Drop all the data except the column you are interested in: in this case, the `location` column.
+
+This example uses the [`unique()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/selectors/unique/), removes the grouping, then uses the [`count()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/count/).
```

Comment:
Use the [`unique()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/selectors/unique/) to return unique values in a specified column. Use the [`count()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/count/) to count the number of unique values.

The following examples uses the Sample NOAA weather data to identify and count unique locations data was collected from. It first uses `group()` to ungroup data and return results in a single table.